apps/status-api/README.md

```diff
@@ -1,6 +1,6 @@
 # DeFiChain Status API
 
-DeFiChain Status API, providing the statuses of different DeFiChain services
+DeFiChain Status API, providing the statuses of different DeFiChain services.
```

Comment:
The changes in this file have both statuses and status. 
I think it should be either all statuses or all status.
I may miss the context, it is your call anyway to change this.

Reply:
Yeah, it is a misunderstanding. `Defichain Status API` provides the statuses of Ocean and blockchain services.